service/worker/scheduler/activities.go

```diff
@@ -294,7 +295,9 @@ func translateError(err error, msgPrefix string) error {
 	if common.IsServiceTransientError(err) || common.IsContextDeadlineExceededErr(err) {
 		return temporal.NewApplicationErrorWithCause(message, errType(err), err)
 	}
-	return temporal.NewNonRetryableApplicationError(message, errType(err), err)
+
+	errorType := util.ErrorType(err)
```

Comment:
use this in both places and remove errType

Reply:
done